src/main/java/ex2/Ex2sol.kt

```diff
@@ -0,0 +1,8 @@
+package ex2
+
+fun isCorrectCapitalization(word: String?): Boolean {
+    val regex = "(^[A-Z][A-Z]+)||(^[A-Z][a-z]+)||(^[a-z][a-z]+)"
```

Comment:
too complicated, we can simplify it. But definitely, regex is the best approach here.

Reply:
simplified a bit to val regex = "(^[A-Z]+)||(^[A-Z][a-z]+)||(^[a-z]+)"